ui/pages/extension/AccessMesheryModal.js

```diff
@@ -0,0 +1,106 @@
+import React, {
+} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography, Grid, Button, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+const styles = makeStyles((theme) => ({
+  paper : {
+    position : 'absolute',
+    width : "30rem",
+    height : "20rem",
+    backgroundColor : theme.palette.background.paper,
+    boxShadow : theme.shadows[5],
+    borderRadius : "4px",
```

Comment:
Use `rem`. Why is the styling not part of modal component?

Reply:
Fixed, now, it is.